cmd/admin-rpc-client.go

```diff
@@ -146,9 +167,58 @@ func sendServiceCmd(cps adminPeers, cmd serviceSignal) {
 		wg.Add(1)
 		go func(idx int) {
 			defer wg.Done()
-			errs[idx] = invokeServiceCmd(remotePeers[idx], cmd)
+			// we use idx+1 because remotePeers slice is 1 position shifted w.r.t cps
+			errs[idx+1] = invokeServiceCmd(remotePeers[idx], cmd)
 		}(i)
 	}
 	wg.Wait()
 	errs[0] = invokeServiceCmd(cps[0], cmd)
 }
+
+func fetchLocks(cps adminPeers, bucket, prefix string, relTime time.Duration) ([]VolumeLockInfo, error) {
```

Comment:
listLocksPeers doesn't sound right. I shall add a comment and change cps to something more meaningful.